modules/ROOT/pages/create-or-modify-a-page.adoc

```diff
@@ -120,12 +120,12 @@ The preview displays the element as it would be displayed on the selected type o
 
 == Export an element
 
-You can export a page or a layout to deploy it in Bonita Portal as a custom page. +
+You can export a page or a layout to deploy it in Bonita Admin Application as a custom page. +
 You can export any element to import it into another UI Designer. +
 
 To export an element, click the *_Export_* button image:images/images-6_0/pb-export.png[Export button] on the Page editor or the UI Designer home page. A zip file is downloaded to your computer. It contains a Bonita custom page, which also suitable for import into another UI Designer. +
```

Comment:
which also suitable =>  which is also suitable

Reply:
Added